NASA-AMMOS/aerie-ui
# Zooming improvements

resolves #826

To test:
1. Create a plan and add some directives and run the simulation to populate the resources
2. Right click anywhere on a resource
3. Hover over the new "Zoom" sub menu
4. Choose a zoom level
5. Verify that the time range centers on where you right clicked and that the time range is now the zoom level that was chosen
